CustomUserNamePasswordValidator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.ServiceModel;
+
+namespace DesktopServer
+{
+    internal class CustomUserNamePasswordValidator : System.IdentityModel.Selectors.UserNamePasswordValidator
```

Comment:
This file and the Core file need to be in the relevant project folders and not at the root.